src/app/shared/utils/consts.ts

```diff
@@ -60,8 +61,9 @@ export const Consts = {
         'ibm-cos': "IBM COS",
         'yig': "YIG Ceph",
         'alibaba-oss' : "Alibaba Object Storage",
-        'aws-file' : 'AWS File Service',
-        'azure-file' : 'Azure File Service'
+        'aws-file' : 'AWS File Storage',
+        'azure-file' : 'Azure File Storage',
+        'aws-block' : 'AWS Block Storage'
```

Comment:
gcp-file name not defined?

Reply:
That is part of the GCP file share PR. This PR only is only for the Block storage service.

Reply:
But in CLOUD_TYPE you have defined the gcp-file. So half part is here

Reply:
:facepalm: let me check and remove that.
i have pushed the changes as part of #414 .